index.html

```diff
@@ -1989,18 +1980,14 @@ <h4>Presentations Using Derived Credentials</h4>
           <p>
 Some zero-knowledge cryptography schemes might enable <a>holders</a> to
 indirectly prove they hold <a>claims</a> from a <a>verifiable credential</a>
-without revealing the <a>verifiable credential</a> itself. In these schemes, a
-<a>claim</a> from a <a>verifiable credential</a> might be used to derive a
-presented value, which is cryptographically asserted such that a <a>verifier</a>
-can trust the value if they trust the <a>issuer</a>.
+without revealing the entire <a>verifiable credential</a>. In these schemes, a
+<a>verifiable credential</a> might be used to derive presentable data, which is
+cryptographically asserted such that a <a>verifier</a> can trust the value if
+they trust the <a>issuer</a>.
           </p>
-
           <p>
-For example, a <a>verifiable credential</a> containing the <a>claim</a>
-<code>date of birth</code> might be used to derive the presented value
-<code>over the age of 15</code> in a manner that is cryptographically
-<a>verifiable</a>. That is, a <a>verifier</a> can still trust the derived value
-if they trust the <a>issuer</a>.
+Some selective disclosure schemes can share only a subset of <a>claims</a>
```

Comment:
```suggestion
Some selective disclosure schemes can share a subset of <a>claims</a>
```

"only" feels a bit strange in the sentence to me... "only" could imply that it's a limitation instead of an advantage (which it is).